[2025-07-25T11:11:32.668+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-25T11:11:32.688+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [queued]>
[2025-07-25T11:11:32.706+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [queued]>
[2025-07-25T11:11:32.707+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-25T11:11:32.740+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract_task.load_db_to_parquet> on 2025-07-11 00:00:00+00:00
[2025-07-25T11:11:32.754+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'elt_to_dwh', 'extract_task.load_db_to_parquet', 'scheduled__2025-07-11T00:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/elt_to_dwh.py', '--cfg-path', '/tmp/tmpctoaxgkt']
[2025-07-25T11:11:32.757+0000] {standard_task_runner.py:105} INFO - Job 79: Subtask extract_task.load_db_to_parquet
[2025-07-25T11:11:32.763+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2832) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-25T11:11:32.764+0000] {standard_task_runner.py:72} INFO - Started process 2842 to run task
[2025-07-25T11:11:32.818+0000] {task_command.py:467} INFO - Running <TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [running]> on host 74cfddf97016
[2025-07-25T11:11:32.904+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='elt_to_dwh' AIRFLOW_CTX_TASK_ID='extract_task.load_db_to_parquet' AIRFLOW_CTX_EXECUTION_DATE='2025-07-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-11T00:00:00+00:00'
[2025-07-25T11:11:32.905+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-25T11:11:32.948+0000] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/scripts/elt_to_dwh/load/load_db_to_parquet.py", line 21, in load_db_to_parquet
    df = pl.read_database(query=query, connection=conn)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/polars/io/database/functions.py", line 249, in read_database
    return cx.execute(
           ^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/polars/io/database/_executor.py", line 501, in execute
    result = cursor_execute(query, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.SyntaxError: syntax error at or near "WHERE"
LINE 27: WHERE
         ^

[2025-07-25T11:11:32.959+0000] {taskinstance.py:1226} INFO - Marking task as FAILED. dag_id=elt_to_dwh, task_id=extract_task.load_db_to_parquet, run_id=scheduled__2025-07-11T00:00:00+00:00, execution_date=20250711T000000, start_date=20250725T111132, end_date=20250725T111132
[2025-07-25T11:11:32.980+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-25T11:11:32.981+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 79 for task extract_task.load_db_to_parquet (syntax error at or near "WHERE"
LINE 27: WHERE
         ^
; 2842)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/scripts/elt_to_dwh/load/load_db_to_parquet.py", line 21, in load_db_to_parquet
    df = pl.read_database(query=query, connection=conn)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/polars/io/database/functions.py", line 249, in read_database
    return cx.execute(
           ^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/polars/io/database/_executor.py", line 501, in execute
    result = cursor_execute(query, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.SyntaxError: syntax error at or near "WHERE"
LINE 27: WHERE
         ^

[2025-07-25T11:11:33.020+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-07-25T11:11:33.068+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-25T11:11:33.179+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-07-25T11:50:55.009+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-25T11:50:55.023+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [queued]>
[2025-07-25T11:50:55.033+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [queued]>
[2025-07-25T11:50:55.034+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-25T11:50:55.048+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract_task.load_db_to_parquet> on 2025-07-11 00:00:00+00:00
[2025-07-25T11:50:55.057+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'elt_to_dwh', 'extract_task.load_db_to_parquet', 'scheduled__2025-07-11T00:00:00+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/elt_to_dwh.py', '--cfg-path', '/tmp/tmp7pq7g3ig']
[2025-07-25T11:50:55.059+0000] {standard_task_runner.py:105} INFO - Job 149: Subtask extract_task.load_db_to_parquet
[2025-07-25T11:50:55.061+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4570) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-25T11:50:55.062+0000] {standard_task_runner.py:72} INFO - Started process 4580 to run task
[2025-07-25T11:50:55.103+0000] {task_command.py:467} INFO - Running <TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [running]> on host 74cfddf97016
[2025-07-25T11:50:55.173+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='elt_to_dwh' AIRFLOW_CTX_TASK_ID='extract_task.load_db_to_parquet' AIRFLOW_CTX_EXECUTION_DATE='2025-07-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-11T00:00:00+00:00'
[2025-07-25T11:50:55.174+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-25T11:50:56.200+0000] {logging_mixin.py:190} INFO - [Company extract] retrieved 23670 rows from the database
[2025-07-25T11:50:56.250+0000] {logging_mixin.py:190} INFO - Saved data from database to parquet successfully at /opt/airflow/data/parquet/companies/crawl_companies-20250711.parquet
[2025-07-25T11:50:56.251+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-25T11:50:56.258+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-25T11:50:56.260+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=elt_to_dwh, task_id=extract_task.load_db_to_parquet, run_id=scheduled__2025-07-11T00:00:00+00:00, execution_date=20250711T000000, start_date=20250725T115055, end_date=20250725T115056
[2025-07-25T11:50:56.322+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-25T11:50:56.347+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-25T11:50:56.357+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-07-26T04:57:08.223+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-26T04:57:08.234+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [queued]>
[2025-07-26T04:57:08.242+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [queued]>
[2025-07-26T04:57:08.242+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-26T04:57:08.255+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract_task.load_db_to_parquet> on 2025-07-11 00:00:00+00:00
[2025-07-26T04:57:08.261+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'elt_to_dwh', 'extract_task.load_db_to_parquet', 'scheduled__2025-07-11T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/elt_to_dwh.py', '--cfg-path', '/tmp/tmppmml1y5f']
[2025-07-26T04:57:08.263+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask extract_task.load_db_to_parquet
[2025-07-26T04:57:08.264+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=676) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T04:57:08.265+0000] {standard_task_runner.py:72} INFO - Started process 686 to run task
[2025-07-26T04:57:08.293+0000] {task_command.py:467} INFO - Running <TaskInstance: elt_to_dwh.extract_task.load_db_to_parquet scheduled__2025-07-11T00:00:00+00:00 [running]> on host 3bc71adbf7e0
[2025-07-26T04:57:08.352+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='elt_to_dwh' AIRFLOW_CTX_TASK_ID='extract_task.load_db_to_parquet' AIRFLOW_CTX_EXECUTION_DATE='2025-07-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-11T00:00:00+00:00'
[2025-07-26T04:57:08.353+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-26T04:57:09.117+0000] {logging_mixin.py:190} INFO - [Company extract] retrieved 23670 rows from the database
[2025-07-26T04:57:09.157+0000] {logging_mixin.py:190} INFO - Saved data from database to parquet successfully at /opt/airflow/data/parquet/companies/crawl_companies-20250711.parquet
[2025-07-26T04:57:09.159+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-26T04:57:09.163+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-26T04:57:09.164+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=elt_to_dwh, task_id=extract_task.load_db_to_parquet, run_id=scheduled__2025-07-11T00:00:00+00:00, execution_date=20250711T000000, start_date=20250726T045708, end_date=20250726T045709
[2025-07-26T04:57:09.202+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-26T04:57:09.215+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-26T04:57:09.216+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
